src/vendor/webauthn.js

```diff
@@ -0,0 +1,185 @@
+"use strict";
```

Comment:
Okay, in that case we have three options:

1. Find a webauth polyfill that has a good license, that we can pull down from NPM, etc
2. Write our own. Seems pretty basic, so this feels like a good option to me. Of course, writing our own means that we have to write unit tests as well, etc. At this point we can get rid of all the things that don't work in IE10/IE11 that gets executed immediately
3. Use this, but find the license on the microsoft page and get clearance from legal

The general idea is that we can't just ship thirdparty code. There are some things to consider:

- What is the license? Where did we get it from? If it's not versioned, how do we upgrade in the future?
- Does the license allow us to modify the code? Again, if we modify the code and there are updates in the future, what is the upgrade process like?